app/controllers/application_controller.rb

```diff
@@ -11,6 +12,24 @@ def cors_preflight
 
   private
 
+  rescue_from "Exception" do |exception|
+    log_error(exception)
+    case exception
+    when VA::API::Common::Exceptions::BaseError
+      render json: { errors: exception.errors }
+    else
+      render json: { errors: VA::API::Common::Exceptions::InternalServerError.new(exception).errors }
+      # FIXME: do we need to re-raise the exception as long as we are logging it and rendering something?
+      # Maybe we will need a new relic specific method to call here??
+      # raise exception if Rails.env.production?
+    end
+  end
+
+  def log_error(exception)
+    Rails.logger.error "#{exception.message}."
+    Rails.logger.error exception.backtrace.join("\n") unless exception.backtrace.nil?
```

Comment:
I think exceptions always have a hash for the backtrace, so not sure they would ever be nil?